src/services/applicationsReviewService.spec.js

```diff
@@ -1,4 +1,6 @@
-import { API_HOSTNAME } from "../config";
+import config from "../config";
+const { API_HOSTNAME } = config;
```

Comment:
does this variable change when the api host changes?

Reply:
The variable itself changes, however the imports are constant (even though the variable in `../config`has changed, the variable inside this file will not update. That's why a refresh is required on change, so that the devtools load the updated value from the localStorage and apply it